### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by danie007**
Added MIT license link for further reference in readme file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,4 +80,4 @@ docker run -it -p 8080:80 --rm opendrinks
 
 ## License
 
-MIT
+The [MIT](https://opensource.org/licenses/MIT) License
```
Comment (alfg) [inline README.md]: Can you link this to the LICENSE.md file instead? The opensource.org license looks like a placeholder.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
-# 🍸 Open Drinks
+🍸 Open Drinks
+===
```
Comment (alfg) [inline README.md]: The formatting looks the same, but can you explain this change, please?

File: README.md (hunk 1 above)
Comment (danie007) [reply README.md]: Yes, I can add it for sure

File: README.md (hunk 2 above)
Comment (danie007) [reply README.md]: The change is very minimal, As I removed the # in-front and added === in the next line. There would not be any changing in format but this is GitHub way of doing things.

**Comment by alfg** [comment]
Closing. Feel free to re-open with requested fixes.

**Pull request closed without merging**